src/main/cpp/sim/run_stride.cpp

```diff
@@ -31,6 +32,10 @@
 #include <string>
 #include <utility>
 
+std::atomic<bool> stride::util::INTERRUPT(false);
```

Comment:
Can you move the definition for `stride::util::INTERRUPT` into `Signals.cpp`, remove its declaration from `ExternalVars.h` and expose it only through a function in `Signals.cpp`? That encapsulate this implementation detail better.